go/vt/vttablet/tabletserver/schema/engine.go

```diff
@@ -165,6 +217,10 @@ func (se *Engine) EnsureConnectionAndDB(tabletType topodatapb.TabletType) error
 
 	log.Infof("db %v created", dbname)
 	se.dbCreationFailed = false
+	// creates _vt schema, the first time the database is created
```

Comment:
Again, I think outside of the package _vt should be an implementation detail that is exposed with the const when others need/want to know.

Reply:
ok